application/src/main/java/org/togetherjava/tjbot/moderation/ModAuditLogWriter.java

```diff
@@ -0,0 +1,111 @@
+package org.togetherjava.tjbot.moderation;
+
+import net.dv8tion.jda.api.EmbedBuilder;
+import net.dv8tion.jda.api.entities.Guild;
+import net.dv8tion.jda.api.entities.TextChannel;
+import net.dv8tion.jda.api.entities.User;
+import net.dv8tion.jda.api.requests.restaction.MessageAction;
+import net.dv8tion.jda.api.utils.AttachmentOption;
+import org.jetbrains.annotations.NotNull;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.togetherjava.tjbot.config.Config;
+
+import java.awt.*;
+import java.io.File;
+import java.nio.charset.StandardCharsets;
+import java.time.temporal.TemporalAccessor;
+import java.util.Optional;
+import java.util.regex.Pattern;
+
+/**
+ * Utility class that allows you to easily log an entry on the mod audit log channel. Thread-Safe.
+ */
+public enum ModAuditLogWriter {
+    ;
+    private static final Color EMBED_COLOR = Color.decode("#3788AC");
+
+    private static final Logger logger = LoggerFactory.getLogger(ModAuditLogWriter.class);
+
+    private static Pattern auditLogChannelNamePattern = null;
+
+    /**
+     * Sends a log embed on the mod audit log channel.
+     *
+     * @param title the title of the log embed
+     *
+     * @param description the description of the log embed
+     *
+     * @param author the user to be added to the embed
+     *
+     * @param timestamp the timestamp to be added to the embed
+     *
+     * @param guild the guild to write this log to
+     *
+     * @param attachments attachments that'll be added to the message
+     */
```

Comment:
Usually we have no newlines between the `@param`s, makes it more readable as well.

Reply:
removed them.